src/main/java/com/amazonaws/kinesisvideo/producer/AutomaticStreamingFlags.java

```diff
@@ -0,0 +1,14 @@
+package com.amazonaws.kinesisvideo.producer;
+
+public enum AutomaticStreamingFlags {
```

Comment:
I moved this from being declared within the ClientInfo class.

Reply:
Nice